include/swift/AST/AnyFunctionRef.h

```diff
@@ -51,6 +51,15 @@ class AnyFunctionRef {
     }
   }
 
+  // Get the natural uncurry level for the embedded function which is the number
```

Comment:
This should use 3 slashes. See the LLVM style guide:

http://llvm.org/docs/CodingStandards.html